### PR TITLE
[Docs] Translate embed/rust to zh

### PR DESCRIPTION
**Pull request by Kodalien**
Signed-off-by: kodalien [3097246093@qq.com](mailto:3097246093@qq.com)
#1097

### CODE REVIEW

**Comment by Kodalien** [comment]
> Signed-off-by: kodalien [kkodalien@gmail.com](kkodalien@gmail.com)
> 
> #1097

**Comment by Kodalien** [comment]
>  Signed-off-by: FuOpikuchu [3097246093@qq.com](mailto:3097246093@qq.com)
> #1097

File: docs/book/zh/src/embed/rust.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # Rust SDK
+
+你可以通过 WasmEdge Rust SDK 将 WasmEdge 嵌入到你的 Rust 应用程序中。
```
Comment (alabulei1) [inline docs/book/zh/src/embed/rust.md:3]: 去掉你的吧

File: docs/book/zh/src/embed/rust.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+WasmEdge Rust SDK 的定义涉及两个 Rust crate， [wasmedge-sys](https://crates.io/crates/wasmedge-sys) 和 [wasmedge-rs](https://crates.io/crates/wasmedge-sdk)。 它们是根据不同的原则和目的设计。 wasmedge-sys crate 定义了一组低级 Rust API，它们简单地包装 WasmEdge C API 并提供安全的对应物，而 wasmedge-rs crate 提供更优雅和符合人体工程学的 API，更适合应用程序开发。
```
Comment (alabulei1) [inline docs/book/zh/src/embed/rust.md:5]: 它们是根据不同的原则和目的设计， 去掉是吧

**Review by alabulei1 — COMMENTED** [review]
Thanks! I have some feedback for you to make the docs read more fluently. Please check out.

**Comment by Kodalien** [comment]
> Thanks! I have some feedback for you to make the docs read more fluently. Please check out.

Thanks! @alabulei1 It has been modified.

**Pull request closed without merging**